### DISCUSSION TITLE
Byte 1 of the 1-byte UTF-8 sequence is invalid.

### DISCUSSION THREAD

**Discussion by ourlang** [nextgenhealthcare/connect#5924] [General]
1. XML data  request mirth connect,xml example （Including Chinese）
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:win="http://winning.amis.org/"><soapenv:Header/><soapenv:Body><win:AmisService><win:callType>UrmsPatientListService.Read</win:callType><win:xmlMessage><![CDATA[<?xml version="1.0" ?><DATASET><TABLE><CARDTYPE>0</CARDTYPE><CARDNO></CARDNO><HZXM>叫号接诊1</HZXM><SFZH></SFZH><BLH></BLH><YYXH></YYXH></TABLE></DATASET>]]></win:xmlMessage></win:AmisService></soapenv:Body></soapenv:Envelope>

2. Middle processing XML strings to JSON

3. Calling third-party interfaces
4
<img width="725" alt="1695020492053" src="https://github.com/nextgenhealthcare/connect/assets/32355185/7310efe2-8ab8-4b78-bf56-be8d344df3b4">

<img width="895" alt="1695020523351" src="https://github.com/nextgenhealthcare/connect/assets/32355185/94dcd617-b4ae-47a8-aa24-3ed793530faf">

**Comment by jonbartels** [comment]
Can you post:
* XML data with formatting https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
* The configuration for your channel source connector? Is UTF-8 enabled?
